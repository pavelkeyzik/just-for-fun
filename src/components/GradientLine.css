.GradientLine__container {
  display: block;
  width: 100%;
  height: 4px;
  background: #333
    linear-gradient(
      90deg,
      #ff69b4 0%,
      #ff69b4 20%,
      #69b4ff 40%,
      #69ffb4 60%,
      #ffb469 80%
    );
}
