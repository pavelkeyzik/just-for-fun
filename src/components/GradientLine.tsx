import React from 'react';

import styles from './GradientLine.module.css';

export function GradientLine(): JSX.Element {
  return <div data-attr="someattr" className={styles.line} />;
}
