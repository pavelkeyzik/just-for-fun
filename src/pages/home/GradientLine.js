import React from 'react';

import './GradientLine.css';

export function GradientLine() {
  return <div className="GradientLine__container">c</div>;
}
