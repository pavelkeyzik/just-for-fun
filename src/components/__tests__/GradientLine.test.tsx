import React from 'react';
import { render } from '@testing-library/react';
import { GradientLine } from '../GradientLine';

test('should render one GradientLine component', () => {
  const { container } = render(<GradientLine />);

  expect(container.firstChild).toMatchSnapshot();
});
