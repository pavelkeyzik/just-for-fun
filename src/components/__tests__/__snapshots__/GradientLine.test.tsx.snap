// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render one GradientLine component 1`] = `
<div
  class="line"
  data-attr="someattr"
/>
`;
